docs/openapi.json

```diff
@@ -82,6 +82,17 @@
               "type": "array"
             }
           },
+          "400": {
+            "description": "Request failed. The incoming payload seems malformed."
+          },
+          "404": {
+            "description": "Request failed. User does not exist.",
+            "examples": {
+              "application/json": {
+                "message": "User 00000000-0000-0000-0000-000000000000 doesn't exist."
```

Comment:
A cosmetic change I suggest is to write in all messages 'does not' and end with a period